data/json/terrain.json

```diff
@@ -3278,6 +3278,51 @@
                 { "item": "splinter", "count": [10, 25] }
             ]
         }
+    },{
+        "type" : "terrain",
+        "id" : "t_tree_coffee",
+        "name": "coffee tree",
+        "description": "This is 'gymnocladus dioicus', or the kentucky coffee tree, which produces roastable coffee pods and brewable leaves in the fall.  If you examined the branches more closely, you could probably find a few mature ones.  You could also cut it down with the right tools.",
+        "symbol": "7",
```

Comment:
Consider adding 
```
"looks_like": "t_tree_plum",
```
so that it will use the plum tree tile in tilesets that don't have a t_tree_coffee tile.